app/src/androidTest/java/com/twilio/video/app/e2eTest/BackgroundSupportTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.twilio.video.app.e2eTest
+
+import androidx.test.ext.junit.rules.activityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiSelector
+import androidx.test.uiautomator.Until
+import com.twilio.video.app.screen.assertRoomIsConnected
+import com.twilio.video.app.screen.assertScreenIsDisplayed
+import com.twilio.video.app.screen.clickDisconnectButton
+import com.twilio.video.app.screen.clickJoinRoomButton
+import com.twilio.video.app.screen.enterRoomName
+import com.twilio.video.app.ui.splash.SplashActivity
+import com.twilio.video.app.util.clearTask
+import com.twilio.video.app.util.retryEspressoAction
+import com.twilio.video.app.util.uiDevice
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class BackgroundSupportTest : BaseUITest() {
+
+    @get:Rule
+    var scenario = activityScenarioRule<SplashActivity>()
+
+    @Test
+    fun it_should_show_a_notification_when_the_app_is_in_the_background() {
+        retryEspressoAction { assertScreenIsDisplayed() }
+
+        enterRoomName("android-test")
```

Comment:
Generate a random string here. It will help prevent room collisions when you have multiple devices/tests running simultaneously.